### PR TITLE
Handle "Mins" and fix other small bug

### PR DESCRIPTION
**Pull request by smthfrmn**
Two bug fixes:

- Fix "mins" issue #9 
- Handle when location is in two timezones (@andreakoelzsch see code comments to confirm it's okay)

### CODE REVIEW

File: RFunction.R [hunk 1]
```diff
@@ -1,444 +1,446 @@
-library('move2')
-library('geodist')
-library('lubridate')
-library('lutz')
-library('sf')
-library('terra')
+library("move2")
```
Comment (smthfrmn) [inline RFunction.R:1]: Most changes were just my automated linter

File: RFunction.R [hunk 2]
```diff
@@ -1,444 +1,446 @@
...
+library("sf")
+library("terra")
 
 # to do:
 # 1. redo all with sf
 # 2. add to output (and csvs): "most common location", i.e. for each cluster cut off at 4 or 5 decimals (round), make table and select location with most occurrences (radius 5-10 or 20-30m should be ok) --> rather with extra radius and hclust (clusters within clusters)
 
-rFunction = function(meth="buff", rad=NULL, dur=NULL, minloc=NULL, dur_unit="days", maxgap=1, gap_unit="days", clu_transm="all", new_dur=24, data, ...) {
-  Sys.setenv(tz="UTC")
+tz_fn <- function(x) {
```
Comment (smthfrmn) [inline RFunction.R:12]: Function for dealing with when a location is in two timezones (just arbitrarily take the first one)

File: RFunction.R [hunk 3]
```diff
@@ -1,444 +1,446 @@
...
+rFunction <- function(meth = "buff", rad = NULL, dur = NULL, minloc = NULL, dur_unit = "days", maxgap = 1, gap_unit = "days", clu_transm = "all", new_dur = 24, data, ...) {
+  Sys.setenv(tz = "UTC")
   time_now <- Sys.time()
-  if (("timestamp" %in% (names(data)))==FALSE) data$timestamp <- mt_time(data)
-  
-  if (is.null(rad))
-  {
+  if (("timestamp" %in% (names(data))) == FALSE) data$timestamp <- mt_time(data)
+
+  dur_udunits <- if (dur_unit != "mins") dur_unit else "min"
```
Comment (smthfrmn) [inline RFunction.R:25]: "mins" fix

File: tests/testthat/test_RFunction.R [hunk 4]
```diff
@@ -1,13 +1,16 @@
-library('move2')
+library("move2")
```
Comment (smthfrmn) [inline tests/testthat/test_RFunction.R:1]: tests

File: tests/testthat/helper.R [hunk 5]
```diff
@@ -1,4 +1,4 @@
 test_data <- function(test_file) {
     test_data_root_dir <- test_path("data")
     readRDS(file = file.path(test_data_root_dir, test_file))
-}
\ No newline at end of file
+}
```
Comment (smthfrmn) [inline tests/testthat/helper.R:4]: nothing

File: RFunction.R [hunk 6]
```diff
@@ -1,444 +1,446 @@
...
+      n.locs <- apply(matrix(cluID), 1, function(x) length(which(result$clusterID == x)))
+      n.ids <- apply(matrix(cluID), 1, function(x) length(unique(mt_track_id(result)[result$clusterID == x])))
+      id.names <- apply(matrix(cluID), 1, function(x) paste(unique(mt_track_id(result)[result$clusterID == x]), collapse = ", "))
+      id.tags <- rep(NA, length(n.ids))
+      if ("tag_local_identifier" %in% names(result)) id.tags <- apply(matrix(cluID), 1, function(x) paste(unique(result[result$clusterID == x, ]$tag_local_identifier), collapse = ", "))
+      # if("tag.local.identifier" %in% names(result)) id.tags <- apply(matrix(cluID), 1, function(x) paste(unique(result[result$clusterID==x,]$tag.local.identifier),collapse=", "))
+
+      # add cumlag.locs (sum of timelags of each location in cluster) - addition from Sep 2022
+      # result$timelag <- unlist(lapply(timeLag(result, units=dur_unit), function(x) c(as.vector(x), NA)))
+      result$timelag <- mt_time_lags(result, units = dur_udunits)
```
Comment (smthfrmn) [inline RFunction.R:337]: actual fix

File: tests/testthat/test_RFunction.R (hunk 4 above)
Comment (andreakoelzsch) [reply tests/testthat/test_RFunction.R:1]: both work

File: RFunction.R (hunk 1 above)
Comment (andreakoelzsch) [reply RFunction.R:1]: what is "linter"?

Comment (smthfrmn) [reply RFunction.R:1]: It's a package that checks the syntax/style of the code automatically to adhere to the language's style guide

I use tidyverse's: https://www.tidyverse.org/blog/2017/12/styler-1.0.0/

**Review by andreakoelzsch — APPROVED** [review]
Thanks a lot for the improvements, Smith! All the format changes make it very difficult to check, what are actually functional changes.

**Pull request merged**